### ISSUE TITLE
[api-extractor] @preapproved does not remove signature of methods from api files

### ISSUE THREAD

**Issue by acchou** [Microsoft/web-build-tools#1113]
The documentation on @preapproved ([link](https://api-extractor.com/pages/tsdoc/syntax/)) is ambiguous about what constructs it can be applied to. In testing it appears that @preapproved doesn't work when there is a public class with internal methods:

```typescript
/** @public */
class C {
  /** @internal @preapproved */
  f() { }
}
```

In this case `C.f()` still appears in the output `.api.ts` file. I can create a separate test case if needed, but I wanted to check to see if this behavior was a known limitation.

**Comment by octogonz** [comment]
`@preapproved` was designed for entire API items that should be omitted from the API review file because reviewers aren't interested in being notified when changes occur. The case you describe might make sense if a single function gets updated frequently and nobody cares about it.

Frankly this feature was not widely used, and there are no test cases for it, so it's probably just an oversight in the logic.

**Comment by octogonz** [comment]
@acchou this was fixed in 7.0.32.  But I recommend using the API Extractor beta is 7.0.38, since it's basically the release candidate.
